modules/serverless-configuring-multi-container-service.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative-serving/config-applications/multi-container-support-for-serving.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-configuring-multi-container-service_{context}"]
+= Configuring a multi-container service
+
+Multi-container support is enabled by default. You can create a multi-container pod by specifiying multiple containers in the service.
+
+.Procedure
+
+. Modify your service to include additional containers. Only one container can handle requests, so specify `port` for exactly one container. Here is an example configuration with two containers:
```

Comment:
@rh-max, what's `port`, isn't this `ports`?

Reply:
@jboxman-rh good catch, fixed